builtins/amp-img.js

```diff
@@ -184,6 +197,22 @@ export class AmpImg extends BaseElement {
     }
   }
 
+  /**
+   * Checks to see if there is a placeholder that is blurred
+   * @private
+   */
+  checkBlur_() {
+    const placeholder = this.getPlaceholder();
+    if (placeholder) {
+      const classes = placeholder.getAttribute('class').split(' ');
+      if (classes.find(function(el) {
+        return el == 'i-amphtml-blur';
+      })) {
+        this.hasBlurredPlaceHolder_ = true;
```

Comment:
Let's add the code that actually uses this boolean. This PR is still quite small.

Reply:
So just to clarify, that would mean add the code to trigger the loading fade-in animation when the image is loaded, right?Or is it something else?

Reply:
Yes.